### PR TITLE
Fixed issues with setup of itest/sigar/sqllogictest

### PR DESCRIPTION
**Pull request by matriv**

### CODE REVIEW

File: blackbox/sqllogictest/src/tests.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ def test_suite():
         port=CRATE_HTTP_PORT,
         transport_port=CRATE_TRANSPORT_PORT,
         settings={
-            'stats.enabled': True,
+            'stats.enabled': 'true',
```
Comment (mfussenegger) [inline blackbox/sqllogictest/src/tests.py]: why is this necessary?

File: blackbox/docs/src/crate/tests.py [hunk 2]
```diff
@@ -118,6 +118,7 @@ def java_transform(s):
 class ConnectingCrateLayer(CrateLayer):
 
     def __init__(self, *args, **kwargs):
+        kwargs['settings'] = {};
```
Comment (mfussenegger) [inline blackbox/docs/src/crate/tests.py]: aren't the settings always there?

File: blackbox/sqllogictest/src/tests.py (hunk 1 above)
Comment (matriv) [inline blackbox/sqllogictest/src/tests.py]: because es complains that this is not a valid boolean value when tries to parse it from string

File: blackbox/docs/src/crate/tests.py (hunk 2 above)
Comment (matriv) [inline blackbox/docs/src/crate/tests.py]: Nope:
https://github.com/crate/crate/pull/4801/files#diff-0fa31503051210744ee04c174af5ddb6L160

Should I add a check: if kwargs is None: ?

File: blackbox/sqllogictest/src/tests.py (hunk 1 above)
Comment (mfussenegger) [inline blackbox/sqllogictest/src/tests.py]: Can we stringify this in the test-layer somehow? Otherwise this is also a breaking change.

File: blackbox/docs/src/crate/tests.py (hunk 2 above)
Comment (mfussenegger) [inline blackbox/docs/src/crate/tests.py]: `kwargs.setdefault('settings', {})`

File: blackbox/sqllogictest/src/tests.py (hunk 1 above)
Comment (matriv) [inline blackbox/sqllogictest/src/tests.py]: You mean something like that?

```
if further_settings:
            for key in further_settings:
                print(key)
                print(further_settings[key])
                if type(key) == bool:
                    if (further_settings[key] == True):
                        further_settings[key] = 'true'
                    else:
                        further_settings[key] = 'false'
```
